### ISSUE TITLE
CONTENT: Additional copy for hood.ie/community

### ISSUE THREAD

**Issue by renrutnnej** [hoodiehq/editorial#68]
Part of https://github.com/hoodiehq/hood.ie/issues/234

### :octocat: Starter Issue

We would like to take the opportunity to invite someone who did not yet contribute to Hoodie to work on this issue. Follow [@YourFirstPR](https://twitter.com/yourfirstPR) on Twitter for more of these issues across many open source projects :relaxed:

---

### :pencil2: The Task

In preparation for the Camp Release we are trying to update the hood.ie site. This specific writing task involves adding content to the [Hoodie community page](http://hood.ie/community/) to update our current contributors list. :bear: 

- [x] Claim this issue (@jo-porter)
- [x] Investigate/interview and then write a one-sentence bio for the following folks: 
 - [x] Charlotte Spencer @Charlotteis 
 - [x] Christoph Witzko @christophwitzko 
 - [x] Nick Herr @HipsterBrown 
 - [x] Jenn Turner @renrutnnej
 - [x] Julia Simplicio @jsimplicio 
 - [x] Patricia Garcia @patriciagarcia 
- [x] Submit to @renrutnnej @Charlotteis or @janl for review (Google docs probably work best for this sort of thing)
- [x] You are done :clap: comment below that your content is ready for review :tada:

Ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq/) if you have any questions :)

**Comment by jo-porter** [comment]
Hallo :) A couple of the listed folks have already been added to the community page, and I've got the everything ready for the remaining content.

I was wondering whether it'd be super cheeky if I took this editorial issue as well, just so I can get the updates done.

Such apologies for my eagerness, when I get into a task there's no stopping me :-/ If you'd rather let another hood.ie newbie tackle this though, that's cool too :)

**Comment by janl** [comment]
@jo-porter GO GO GO

**Comment by renrutnnej** [comment]
Hi @jo-porter! No apology necessary, thank you for the eagerness!

**Comment by Charlotteis** [comment]
@jo-porter Hi! I ticked off the bios for myself and julia as we already did those.

I cannot see a world in which you are being cheeky, please feel free to contribute! :D

**Comment by jo-porter** [comment]
Thanks all :)

**Comment by jo-porter** [comment]
Is email going to be the simplest way to contact folks for copy, do we think?

**Comment by janl** [comment]
or @-mention them here :)

**Comment by janl** [comment]
email sounds fine though.

Haunting on Slack is also acceptable ;)

**Comment by renrutnnej** [comment]
Hey @jo-porter, just checking in to see how you're doing with this issue. If you need any help, or are blocked, please let me know :relaxed:

**Comment by jo-porter** [comment]
Hey @renrutnnej, 

So, all the content has been added, there's a pull request in hoodiehq/hood.ie relating to it. I also cropped the photos and will create a pull request for that in hoodie-css - I'm not sure if what I've done is exactly right though - sorry! There were a couple of things that @gr2m pointed out needed to be fixed, and they've been done. I thought I messaged Gregor to let him know, but I can't seem to see that message now - sorry Gregor!
